provision/testdata/cloud_tar/cluster-network-03-config.yaml

```diff
@@ -6,7 +6,9 @@ metadata:
 spec:
   disableMultiNetwork: true
   clusterNetwork:
-  - cidr: "14.3.0.1/16"
+  - cidr: [
+    "14.3.0.1/16"
+]
     hostPrefix: 23
```

Comment:
Below change is required for dualstack:
cidr should not be list for dualstack case, it should be another object of clusterNetwork like below.
clusterNetwork:
  - cidr: 14.3.0.1/16
    hostPrefix: 23
  - cidr: 2001:fb8:42::/56
    hostPrefix: 64